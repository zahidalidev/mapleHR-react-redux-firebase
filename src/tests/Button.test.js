import { shallow } from 'enzyme'

import Button from 'components/button'

const validButton = <Button btnName='Login' />

describe('Button', () => {
  test('renders', () => {
    expect(shallow(validButton).exists()).toBe(true)
  })

  test('should render button', () => {
    expect(shallow(validButton).getElements()).toMatchSnapshot()
  })

  test('should be an button element', () => {
    const wrapper = shallow(validButton)
    const button = wrapper.find('ForwardRef(Button)')
    expect(button.exists()).toBe(true)
  })
})
