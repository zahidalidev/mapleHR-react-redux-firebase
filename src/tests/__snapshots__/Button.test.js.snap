// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render button 1`] = `
Array [
  <div
    className="sign-in-form-button"
  >
    <ForwardRef(Button)
      className="landing-page-button"
      data-flag="Login"
      style={
        Object {
          "backgroundColor": "#00b8b0",
        }
      }
      type="submit"
      variant="contained"
    >
      Login
    </ForwardRef(Button)>
  </div>,
]
`;
